src/Monitors/IMUMonitor.cpp

```diff
@@ -31,12 +31,21 @@ void IMUMonitor::IMU_init()
 
 void IMUMonitor::execute()
 {
+    // handle latent turn on / turn off variables
+    if (sfr::imu::turn_off == true && sfr::imu::powered == false) {
+        sfr::imu::turn_off = false;
+    }
+    if (sfr::imu::turn_on == true && sfr::imu::powered == true) {
+        sfr::imu::turn_on = false;
+    }
+
     if (sfr::imu::turn_on == true && sfr::imu::powered == false) {
 #ifdef VERBOSE
         Serial.println("turned on IMU");
 #endif
         IMUMonitor::IMU_init();
         if (sfr::imu::init_mode == (uint16_t)sensor_init_mode_type::complete) {
+            sfr::imu::turn_on = false;
```

Comment:
Is there a reason why turn_on is set to false here? I don't think the code ensures that the imu is turned on if the sensor_init_mode_type is complete.

Reply:
turn on represents if we want to turn on the imu on or off - not if it is actually powered

Reply:
ok, so we don't want to turn on the imu anymore after we complete the initialization. Makes sense. Thank you Lauren!